source/concepts/faq.md

```diff
@@ -66,12 +66,13 @@ Nixpkgs and NixOS have both stable and rolling releases.
 
 ### Stable
 
-- On Linux (including NixOS and WSL), use [`nixos-*`](https://github.com/NixOS/nixpkgs/branches/all?query=nixos-).
+- On macOS/Darwin, use [nixpkgs-*-darwin](https://github.com/NixOS/nixpkgs/branches/all?query=nixpkgs-)
```

Comment:
Darwin is still a minority of our users, it doesn't really make sense to put this as the first item.